.github/CODEOWNERS

```diff
@@ -0,0 +1,9 @@
+# iOS
+/platform/ios/ @mapbox/maps-ios-reviewers
+
+# macOS
+/platform/macos/ @1ec5 @mapbox/maps-ios-reviewers
+
+# Darwin
+/platform/darwin/ @mapbox/maps-ios-reviewers
```

Comment:
platform/darwin/ affects macOS just as much as iOS, so I might want to be notified for this path, or maybe I should join the maps-ios-reviewers team. 🤷‍♂️

Reply:
Happy to do either - was trying to avoid extra notifications for you 😉